konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/pipeline/SequencePipelineExecutor.java

```diff
@@ -66,7 +69,13 @@ public Data exec(Data data) {
         Context ctx = null; //TODO
         Data current = data;
         for(PipelineStepRunner psr : runners){
+            if (profiler.profilerEnabled()) {
+                profiler.eventStart(psr.toString());
+            }
             current = psr.exec(ctx, current);
+            if (profiler.profilerEnabled()) {
+                profiler.eventEnd(psr.toString());
```

Comment:
For the profile name:
(a) Add a PipelineStep.name() method:
```
    default String name(){
        return getClass().getSimpleName();
    }
```

(b) use `profiler.eventStart(psr.getPipelineStep().name());` instead